### DISCUSSION TITLE
Integratore non funzionante

### DISCUSSION THREAD

**Discussion by Bilo2110** [aturri/ha-zcsazzurro#19] [Q&A]
Ciao, da qualche giorno l'integrazione non funziona correttamente.. nonostante il logger comunichi correttamente con il Cloud zucchetti (l'app azzurron monitoring funziona correttamente, anche l'app solarman e l'integrazione solarman) la tua integrazione non mi funziona correttamente. I dati sono in ritardo e a volte non si aggiornano.
Esempi:
Batteria ferma a 95% per parecchie ore
Energia immessa in rete ferma a 40w sia in ora di massimo scambio

HA aggiornato all'ultima versione.

Ho già provato a disinstallare la tua integrazione e installarla nuovamente ma nulla

**Comment by aturri** [comment]
Ciao,
controlla qui: https://github.com/aturri/ha-zcsazzurro/issues/17

**Comment by Bilo2110** [comment]
Grazie, quindi è un problema del server di terze parti..
Hai avuto modo di contattarli per sapere il perché di questo problema?
________________________________

**Reply by g4box** [reply]
Sono nella stessa situazione spero si possa risolvere.
